deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -119,6 +119,7 @@ def GenerateConfig(context):
 # Ubuntu update.
 sudo apt-get update -y
 sudo apt-get upgrade -y
+sudo apt-get update && sudo apt-get --assume-yes install google-cloud-sdk
```

Comment:
Can we separate this change into another PR, since it's not directly related to switching to the image version?  Generally speaking, it's good practice to do this separation so that any problems are easier to troubleshoot and find.

Reply:
This is a problem only occurs in image version 18.04. When the startup script of the VM runs, it doesn't have google cloud sdk library installed (e.g. complaining gsutil command not found) that's why we need to add this line here to install the google cloud sdk library before the rest of the installation beings.

Reply:
Ah, okay, that makes sense then.  LGTM!